invenio_search/api.py

```diff
@@ -74,20 +74,17 @@ def search(self, user_info=None, collection=None, enhance=True, **kwargs):
             query.accept(walker)
             query = {
                 "multi_match": {
-                    "message": {
-                        "query": self._query,
-                        "operator": "or",
-                        "zero_terms_query": "all",
-                        "fields": [
-                            "title^3",
-                            "title.raw^10",
-                            "abstract^2",
-                            "abstract.raw^4",
-                            "author^10",
-                            "author.raw^15",
-                            "reportnumber^10",
-                            "eprint^10",
-                            "doi^10"]}}}
+                    "query": self._query,
```

Comment:
Indeed my mistake here :disappointed: I didn't check the empty query. 

I think the `or` operator is the default since 'Heavy gravitino banana` and `Heavy gravitino` both return 26 results